### ISSUE TITLE
Copy button implementation moves focus away from the button when activating it via keyboard

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#1327]
## Description

Using the attribution copy button via keyboard moves the focus away from it to the temporary element used by ClipboardJS. This messes up focus management for keyboard users pretty badly!

## Reproduction

1.  Focus the copy button with your keyboard
2.  Type "Enter" to activate the button and copy the text
3.  Note that the button has already lost its focus styles.
4. Type "Tab" and see that the last related result is focused instead of anything logically near the copy button.

## Additional context
Found while reviewing https://github.com/WordPress/openverse-frontend/pull/1316

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by ramadanomar** [comment]
I would be interested in resolving this bug.

**Comment by sarayourfriend** [comment]
@ramadanomar thanks, I've assigned the issue to you. Feel free to ping if you need any help.

**Comment by ramadanomar** [comment]
I was able to reproduce this bug last night using Chromium on Ubuntu but fail to reproduce it on my laptop (Macbook). Also more elements were focusable on Chromium than on Safari.

[See slack for recording.](https://wordpress.slack.com/archives/C02012JB00N/p1660297296896499)

**Comment by sarayourfriend** [comment]
Can you check that you've got the "keyboard navigation" feature enabled in macOS? It behaves strangely when this is not enabled: https://www.a11yproject.com/posts/macos-browser-keyboard-navigation/
